psi4/src/psi4/dlpno/mp2.h

```diff
@@ -129,7 +129,15 @@ class DLPNOMP2 : public Wavefunction {
     SparseMap riatom_to_bfs2_; ///< which orbital BFs are needed for DF int transform (second index)
 
     // Dense analogues of some sparse maps for quick lookup
+
+    /* Takes an atom index and a global LMO index 
+    to return the sparse LMO index to that atom,
```

Comment:
I find "for that atom" or "on that atom" much clearer than "to that atom." Same comment below.

Ping me on fix - this is the only outstanding issue that I see.

Reply:
Fixed, please proceed with merge